### DISCUSSION TITLE
Explorer to Open In New Tab

### DISCUSSION THREAD

**Discussion by ukmartin** [valinet/ExplorerPatcher#1776] [Ideas]
The latest version of Windows 11 introduced the option to open Explorer in New Tabs alongside the long-time option of opening in a New Window. This gives the option to browse various tabs in the Windows Explorer in the same way as we use web browsers. Personally, I think this is a superb new feature.

Whilst this appears in the Windows 11 (right-click) context menu now, it is lost when Explorer Patcher is applied.

Would it be possible to have "the best of both worlds" and to keep the new "Open In New Tab" as an option that can be selected by those people who like that functionality, when using the old Windows 10 context menus, in a future version of Explorer Patcher please?

**Comment by pyrates999** [comment]
Does this happen when a specific option is enabled?  Or just with the defaults applied?

Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Check if explorer has tabs.

Then apply your settings again in explorer patcher to see if it fixes it.  And then check for what option removed the tabs in explorer.

**Reply by oysmik** [reply]
Still does not work, I'm on 22631.2129

**Reply by pyrates999** [reply]
Then explorer patcher is not causing this since you tested it without explorer patcher installed.

You are also running an insider build: https://blogs.windows.com/windows-insider/2023/08/02/announcing-windows-11-insider-preview-build-22621-2129-and-22631-2129/

They could have bugs in them since they are not fully tested by Microsoft.

**Reply by oysmik** [reply]
I have EP installed, just reinstalled now, not an idiot.
But yes, I'm running Win Insider (Beta Channel) - might be the issue here.

**Reply by pyrates999** [reply]
Unless you like testing early software and are ok with it containing bugs, I would not run insider builds, nor would I install the preview updates.

**Comment by ukmartin** [comment]
Hi
I use Windows 11 version 22H2. The Open in New Tab option was a feature introduced by Microsoft in this version last October (https://www.xda-developers.com/windows-11-2022-update-file-explorer-guide/)  Yes, it appears to have been on Insider versions before that too, but I don't subscribe to that, and just get the standard Windows Updates as and when officially released.

You may recall, that I had to uninstall Explorer Patcher due to some issues I was having (https://github.com/valinet/ExplorerPatcher/discussions/1761#discussioncomment-6632122)  at which point I was using Windows 11 until I reinstalled EP per your recommended method. During that time of using the basic Win 11, I got to really like the Open in New Tab option, and would really think that it would be a benefit as an option for a future EP version too.

**Reply by ukmartin** [reply]
Ii should just clarify, that since last October, I've been using Explorer Patcher, which had the problems that were solved on the other thread, then I used basic Windows 11 for a while, and now I'm back to Explorer Patcher.  I can confirm that Open in New Tab is only available on Windows 11 if EP is not or un-installed. Even with the fresh installation of the latest version of EP, that I did a couple of days ago, the option has not been inherited into my current configuration.

**Reply by pyrates999** [reply]
What version and build did you do a fresh installation from?

You should test this from this version: https://support.microsoft.com/en-au/topic/july-11-2023-kb5028185-os-build-22621-1992-605fa18f-bd49-41d8-80b1-245080e26c3d

That is the current release.  If it is happening there, then you have a reason for it to be fixed.
